UI/PauseScreen.cpp

```diff
@@ -447,7 +447,7 @@ void GamePauseScreen::CreateViews() {
 		playButton_->SetImageID(g_Config.bRunBehindPauseMenu ? ImageID("I_PAUSE") : ImageID("I_PLAY"));
 		return UI::EVENT_DONE;
 	});
-	rightColumnHolder->Add(new Spacer(10.0f));
+	//rightColumnHolder->Add(new Spacer(10.0f));
```

Comment:
Hm, I feel like it's better to keep the space?  But otherwise should just delete this line, right?

-[Unknown]

Reply:
Reverted.